src/tasks/installers/olm.ts

```diff
@@ -313,43 +318,61 @@ export class OLMTasks {
           task.title = `${task.title}: ${ctx.isPreInstalledOLM}...OK`
         },
       },
+      {
+        title: `Check if operator is installed in ${DEFAULT_OPENSHIFT_OPERATORS_NS_NAME} namespace`,
+        task: async (ctx: any, task: any) => {
+          if (await kube.operatorSubscriptionExists(SUBSCRIPTION_NAME, DEFAULT_OPENSHIFT_OPERATORS_NS_NAME)) {
+            ctx.operatorNamespace = DEFAULT_OPENSHIFT_OPERATORS_NS_NAME
+            task.title = `${task.title}...Found`
+          }
+          task.title = `${task.title}...Not Found`
+        },
+      },
       {
         title: `Delete(OLM) operator subscription ${SUBSCRIPTION_NAME}`,
         enabled: ctx => ctx.isPreInstalledOLM,
-        task: async (_ctx: any, task: any) => {
-          await kube.deleteOperatorSubscription(SUBSCRIPTION_NAME, flags.chenamespace)
+        task: async (ctx: any, task: any) => {
+          const checlusters = await kube.getAllCheClusters()
+          if (kube.operatorSubscriptionExists(SUBSCRIPTION_NAME, DEFAULT_OPENSHIFT_OPERATORS_NS_NAME) && checlusters.length === 0) {
+            await kube.deleteOperatorSubscription(SUBSCRIPTION_NAME, ctx.operatorNamespace)
```

Comment:
Sorry, but you completely removed case with delete operator subscription for one specific namespace. The same for another delete tasks....